charts/argo-cd/templates/NOTES.txt

```diff
@@ -142,7 +142,7 @@ In order to access the server UI you have the following options:
 {{ if eq (toString (index (coalesce .Values.server.config .Values.configs.cm) "admin.enabled")) "true" -}}
 After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running:
 
-kubectl -n {{ .Release.Namespace }} get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
+kubectl -n {{ .Release.Namespace }} get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d | xargs
```

Comment:
What what wbe the reasoning and use case for adding `xargs` here?

Reply:
Thank you for replying. It is just a formatting thing. `xargs` adds a new line character so that the bash prompt shows in the new line. I always use it for better visibility.